### DISCUSSION TITLE
Characters with diacritics/accents are removed from tags. Feature or bug?

### DISCUSSION THREAD

**Discussion by xax-git** [akosbalasko/yarle#621] [General]
Czech, Slovak, Polish ... special characters with diacritics are removed from Tags, but Evernote as well as Obsidian supports it.
Characters like "ě š č ř ž ý á í é" are removed from tags.
Is it possible to configure somewhere Yarle to not do it?

![image](https://github.com/akosbalasko/yarle/assets/9274619/513abccb-5496-489d-81b8-cad7b34e114b)

**Comment by rcsoul** [comment]
I encountered the exact same problem where all the Traditional Chinese characters in the tags were removed.
